src/test/java/gov/healthit/chpl/aqa/pageObjects/ScheduledJobsPage.java

```diff
@@ -0,0 +1,30 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class ScheduledJobsPage definition.
+ */
+public final class ScheduledJobsPage extends BasePage {
+    private ScheduledJobsPage() {
+    }
+
+    public static WebElement typesOfSheduledJobs(final WebDriver driver) {
+        WebElement webelement  = driver.findElement(By.xpath("//table[@class='table table-condensed']//tbody"));
+        return webelement;
+    }
+
+    public static List<WebElement> getRowElements(final WebElement row) {
```

Comment:
Both of these "getRowElements" and "getColElements" look like they should be in the BasePage file; there's nothing on either of these specific to the scheduledJobs page